src/js/internal/debugger.ts

```diff
@@ -20,12 +20,12 @@ export default function (
 
   const { protocol, href, host, pathname } = debug.url;
   if (!protocol.includes("unix")) {
-    console.log(dim("--------------------- Bun Inspector ---------------------"), reset());
-    console.log(`Listening:\n  ${dim(href)}`);
+    console.error(dim("--------------------- Bun Inspector ---------------------"), reset());
+    console.error(`Listening:\n  ${dim(href)}`);
     if (protocol.includes("ws")) {
-      console.log(`Inspect in browser:\n  ${link(`https://debug.bun.sh/#${host}${pathname}`)}`);
+      console.error(`Inspect in browser:\n  ${link(`https://debug.bun.sh/#${host}${pathname}`)}`);
     }
-    console.log(dim("--------------------- Bun Inspector ---------------------"), reset());
+    console.error(dim("--------------------- Bun Inspector ---------------------"), reset());
```

Comment:
console.error turns text red

this should be Bun.stderr.writer() and then writer.write()

Reply:
Ahh, good point! Let me fix that.

Reply:
Hmm. From your last comment I was of the impression that you asked me to do something like 

```
const writer = Bun.stderr.writer();
writer.write('foobar\n')
```

But nothing gets printed with this code. Not sure if its a bug or I cant see the logic. 

In any case, I ended up implementing the same result using

    Bun.write(Bun.stderr, "foobar\n")

Let me know if you feel like it needs more tweaking to be up to standard.